PowerEditor/src/NppNotification.cpp

```diff
@@ -847,6 +847,7 @@ BOOL Notepad_plus::notify(SCNotification *notification)
 			}
 
 			updateStatusBar();
+			updateFunctionList();
 			AutoCompletion * autoC = isFromPrimary?&_autoCompleteMain:&_autoCompleteSub;

```

Comment:
I would use the following code instead of using updateFunctionList();
```
if (_pFuncList && (!_pFuncList->isClosed()) && _pFuncList->isVisible())
    _pFuncList->markEntry();
```

Reply:
Done